api/Document.json

```
@@ -11507,15 +11507,17 @@
               "partial_implementation": true,
               "notes": [
                 "Doesn't fire the <code>visibilitychange</code> event when navigating away from a document, so also include code to check for the <code>pagehide</code> event (which does fire for that case in all current browsers). See WebKit bugs <a href='https://webkit.org/b/116769'>116769</a>, <a href='https://webkit.org/b/151234'>151234</a>, <a href='https://webkit.org/b/151610'>151610</a>, and <a href='https://webkit.org/b/194897'>194897</a>.",
-                "The <code>onvisibilitychange</code> attribute was not supported until Safari 10.1. To listen to this event in earlier versions of Edge, use <code>document.addEventListener('visibilitychange', function() {});</code>."
+                "The <code>onvisibilitychange</code> attribute was not supported until Safari 10.1. To listen to this event in earlier versions of Safari, use <code>document.addEventListener('visibilitychange', function() {});</code>.",
+                "In Safari < 14.0, the event didn't bubble, so only <code>document.addEventListener('visibilitychange', ...)</code> would work, and not<code>window.addEventListener('visibilitychange', ...)</code>."
```

suggestion: we may as well put the more recently applicable notes earlier and ~make these consistent with each other~ actually, on second thought, this `onvisibilitychange` is not really relevant here (it's captured by the compat data for the attribute itself), so we'll just drop it.

```suggestion
                "Before Safari 14, the event does not bubble, so <code>document.addEventListener('visibilitychange', ...)</code> works, but <code>window.addEventListener('visibilitychange', ...)</code> does not."
```